src/platform/site-wide/sass/merger.scss

```diff
@@ -245,7 +242,19 @@
       }
     }
 
+    #login-root {
+      .profile-nav {
+        justify-content: space-between;
+      }
+    }
+
     .vet-toolbar {
+      margin-left: 0;
+      [data-reactroot] {
+        align-items: baseline;
+        display: inherit;
+      }
```

Comment:
Seems like we could use a more descriptive selector here than `[data-reactroot]`.